centreon/lang/de_DE.UTF-8/LC_MESSAGES/help.po

```diff
@@ -1369,8 +1369,8 @@ msgstr ""
 #: centreon/www/include/configuration/configNagios/help.php:500
 msgid ""
 "This is the maximum number of seconds that Monitoring Engine will allow "
```

Comment:
```suggestion
"This is the maximum number of seconds that the monitoring engine will allow "
```

Reply:
the end of the toolitip must be corrected too. (If a service check runs longer than this limit, the monitoring engine will kill it off, thinking it is a runaway process."